ushahidi/platform
# 1236 unique survey titles

This pull request makes the following changes:
- Changed attribute validation
- Added UUID to attribute creation for key

Test these changes by:
- Create a new attribute for a survey
- Create an identically named attribute for a different survey
- Both should save and function correclty

Related to [platform-client/pull/268](https://github.com/ushahidi/platform-client/pull/268)

Fixes ushahidi/platform#1236

Ping @ushahidi/platform

…y on update, added key creation via uuid version 4 to create step


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform/1274)
